FlowCrypt/Functionality/Services/AppStartup.swift

```diff
@@ -77,6 +79,12 @@ struct AppStartup {
         }
     }
 
+    private func getUserOrgRulesIfNeeded() {
+        if DataService.shared.isLoggedIn {
+            _ = OrganisationalRulesService().fetchOrganisationalRulesForCurrentUser()
```

Comment:
@Kharchevskyi  I am not sure that's a good way to fetch anything on startup. Is there any other place that is meant to fetch any initial parameters?

Reply:
For clarity, this one is a remote HTTP call that may potentially take a long time (on lousy connections etc)

Reply:
It's being called asynchronously, so app is not going to wait until it's finished.

Reply:
@ekievsky Please check `AppStartup` -> `initializeApp` I think this is what you are looking for)

Reply:
@Kharchevskyi the method `getUserOrgRulesIfNeeded` is called from `initializeApp`